src/main/java/interface_adapter/delete_clothing_item/DeleteViewModel.java

```diff
@@ -1,13 +1,16 @@
 package interface_adapter.delete_clothing_item;
 
 import interface_adapter.ViewModel;
+import lombok.Getter;
```

Comment:
In Clean Architecture, it's generally acceptable to use tools like Lombok in the outer layer as long as they don't introduce dependencies that would violate the architectural boundaries.